lib/index.js

```diff
@@ -29,6 +30,11 @@ class Client {
     throw "invalid URL argument"
   }
 
+  async put(actionType, data) {
```

Comment:
Do we need `async` here if we aren't using `await` in this function/method?

Reply:
Probably not, let do a double-check with calm.

Reply:
@anuveyatsu  Done, Removed!
The real test result in the comment below.